assets/javascripts/discourse/components/chat-channel-row.js

```diff
@@ -14,13 +12,13 @@ export default Component.extend({
   router: service(),
   chat: service(),
 
-  @discourseComputed("active", "channel.muted")
-  rowClassNames(active, muted) {
-    const classes = ["chat-channel-row"];
+  @discourseComputed("active", "channel.{id,muted}")
```

Comment:
oh wow what. does this mean 

```
@discourseComputed("active", "channel.id", "channel.muted")
```

? Asking because it's new to me.

Reply:
yes indeed

Reply:
but that's probably not going to be useful in future